CMakeLists.txt

```diff
@@ -100,12 +100,15 @@ ExternalProject_Add(ufs-weather-model
   PREFIX ${CMAKE_CURRENT_BINARY_DIR}/ufs-weather-model
   GIT_REPOSITORY https://github.com/climbfuji/ufs-weather-model
   GIT_SUBMODULES_RECURSE TRUE
-  GIT_TAG feature/consolidate_dom_update_20221114  # updated from develop on Mar 6
+  # DH* 20230717 TODO MERGE BACK INTO 20221114 and update date tag
+  #GIT_TAG feature/consolidate_dom_update_20221114  # updated from develop on Mar 6
+  GIT_TAG feature/consolidate_dom_update_20230717  # updated from develop on 2023/07/17
```

Comment:
Note. This needs to be reverted after the update is tested and the branch `feature/consolidate_dom_update_20230717` and all the submodule branches are merged back into the original branches.